addons/survey/views/survey_templates_statistics.xml

```diff
@@ -215,16 +252,36 @@
                     <thead>
                         <tr>
                             <th>User Responses</th>
-                            <th>Occurrence</th>
+                            <th class="col-2">Occurrence</th>
+                            <th t-if="question.is_scored_question" class="col-2">Score</th>
                         </tr>
                     </thead>
                     <tbody>
                         <tr t-foreach="question_data['common_lines']" t-as="common_line">
                             <td>
-                                <span t-if="question.question_type == 'numerical_box'" t-esc="common_line[0]"/>
-                                <span t-if="question.question_type == 'date'" t-esc="common_line[0]" t-options='{"widget": "date"}'/>
+                                <t t-if="question.question_type == 'numerical_box'">
+                                    <t t-set="right_answer" t-value="question.is_scored_question and common_line[0] == question.answer_numerical_box"/>
```

Comment:
I think we could get rid of those "right_answer" variables and make the check directly in the `<i>` element?
Seems they are only used once.

Reply:
There are used in the other column too (see line 280).